README.rst

```diff
@@ -299,6 +299,19 @@ this, you have to pass in ``cssutils_logging_handler`` and
     >>> mylog.getvalue()
     'CSSStylesheet: Unknown @rule found. [2:1: @keyframes]\n'
 
+Choosing the cache implementation
+-----------------------------------
+
+By default, ``premailer`` uses `LFUCache
+<https://cachetools.readthedocs.io/en/latest/#cachetools.LRUCache>`__ to cache
+CSS styles during parsing. But it is possible to choose a different
```

Comment:
It's not just for "CSS styles during parsing", right?

Reply:
Yes right. Updated.